### DISCUSSION TITLE
fix open redirect in examples

### DISCUSSION THREAD

**Discussion by papatie** [maxcountryman/axum-login#243] [Ideas]
## open redirect in examples

`http://localhost:3000/login?next=https%3A%2F%2Fduckduckgo.com%2F`

### Description
The examples code has a open redirect 'security flaw' because the next value is not properly validated. This behavior could also be useful in some cases, but we should at least have a comment in the examples code explaining that we would have some checks or alternative logic in real world code.

for example [here](https://github.com/maxcountryman/axum-login/blob/ddfd0d0398a8a6e55493dc9d12e5a2c632253dc7/examples/sqlite/src/web/auth.rs#L65)

**Comment by maxcountryman** [comment]
This [was a problem in flask-login](https://github.com/maxcountryman/flask-login/commit/cab35692e4f1f0427d840de3f2535514d5c439a8) as well.

It's also possible to address this with the login_required macro, but it's unclear if that's the right place.

As you point out, an application can do its own validation; this might be an acceptable pattern.
